deploy/k8s/overlays/openshift/umami/umami-route.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: route.openshift.io/v1
```

Comment:
I assume this route will allow use to access the umami login page using ui.instructlab.ai:3001 and qa.ui.instructlab.ai:3001?

Reply:
So I did try to make those changes but OpenShift Routes do not directly expose services on custom ports like 3001 to external clients via the host field. To do this we either would have to swap to an ingress or proxy it. I think this is suggestion is good, but as discussed there will be multiple rounds of PRs related to the metrics deployment. Lets make an issue for it and address later.